README.md

```diff
@@ -105,14 +103,13 @@ All your sessions are available on the [AgentOps dashboard](https://app.agentops
 
 Build Crew agents with observability with only 2 lines of code. Simply set an `AGENTOPS_API_KEY` in your environment, and your crews will get automatic monitoring on the AgentOps dashboard.
 
-AgentOps is officially supported on Crew's latest rc branch: `crewai==0.28.9rc1`
-
-* [AgentOps integration example](https://docs.agentops.ai/v1/integrations/crewai)
-* [Offical CrewAI documentation](https://docs.crewai.com/how-to/AgentOps-Observability)
-
+- [AgentOps integration example](https://docs.agentops.ai/v1/integrations/crewai)
```

Comment:
AgentOps is integrated with CrewAI on a pre-release fork. Install crew with `pip install git+https://github.com/AgentOps-AI/crewai.git`